v3-docs/v3-migration-docs/index.md

```diff
@@ -29,7 +29,7 @@ You can upgrade your Meteor 2.x project by running the command below inside your
 
 ```bash
 meteor update --release 3.0-rc.2
-meteor reset #resets local DB and project to a fresh state
+meteor reset #resets project to a fresh state
```

Comment:
I don't think is required in this moment to clean the local database from your project to do the upgrade from v2 to v3. It will when doing it in a future version that is a mismatch between the mongo driver versions.

Should we add the option `--db` here and keep the warn about reset of the local db for also future references?